I_can_help_you_choose_a_new_smartphone/main.js

```diff
@@ -1,2 +1,358 @@
 
-console.log('hello');
\ No newline at end of file
+//-------------------------------random arrey-----------------------------------
+function makeItems() {
+
+  const makes = ['Apple', 'Huawei', 'ASUS', 'Sony', 'Alcatel', 'Motorola', 'Nokia', 'Samsung', 'ZTE', 'Xiaomi'];
+  const price = getRandomPrice(1000, 30000);;
+  const freeDelivery =[false, true];
+
+  let randomMakes = Math.floor(Math.random() * makes.length);
+  let randomDelivery = Math.floor( Math.random() * 2);
+
+  function getRandomPrice(min, max) {
+
+    let num = Math.random() * (max - min) + min;
+    beautifullNum = Math.trunc(num / 100) * 100;
+    return beautifullNum;
+  }
+
+  let item = {
```

Comment:
you can return 'item' straightaway, without creation variable for better performance